modules/@apostrophecms/ui/ui/apos/components/AposSlat.vue

```diff
@@ -114,6 +114,10 @@ export default {
     hasRelationshipSchema: {
       type: Boolean,
       default: false
+    },
+    imageEditor: {
```

Comment:
The editor component name should be passed through, this hardcoded boolean removes the ability to have other custom relationship editors for other types.